javaee/jsonb/src/main/java/org/wildfly/swarm/ts/javaee/jsonb/HelloServlet.java

```diff
@@ -6,12 +6,14 @@
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
+
 import java.io.IOException;
 
-@WebServlet("/")
+@WebServlet("/hello")
```

Comment:
Why not keep this on `/` and put the `AdaptServlet` to `/adapt`, possibly using a query parameter to distinguish the `from` and `to` cases? You wouldn't need to do the `RequestDispatcher` dance.